gateway/src/apicast/threescale_utils.lua

```diff
@@ -187,17 +199,25 @@ function _M.match_xml_element(xml, element, value)
   return string.find(xml, pattern, xml_header_len, xml_header_len, true)
 end
 
--- error and exit
-function _M.error(...)
+-- error without exit
+function _M.error_silently(...)
```

Comment:
I would maybe rename this to `error_gracefully()` because it's not a silent error as we are going to log the failed connection to redis.

Reply:
Noted 📝